count_files/__main__.py

```diff
@@ -171,31 +173,38 @@ def main_flow(*args: [argparse_namespace_object, Union[bytes, str]]):
         loc_text = ' the current directory'
     else:
         location = os.path.expanduser(args.path)
-        loc_text = ':\n' + location
+        loc_text = ':\n' + os.path.normpath(location)
 
     if not os.path.exists(location):
-        parser.exit(status=1, message=f'The path {location} does not exist, or there may be a typo in it.')
+        parser.exit(status=1, message=f'The path {location} '
+                                      f'does not exist, or there may be a typo in it.')
 
     if not include_hidden and is_hidden_file_or_dir(location):
-        parser.exit(status=1, message=f'\nNot counting any files, because {loc_text[2:]} has hidden folders.\n'
-                                      f'Use: python -m count_files {args.path} --all')
+        # skip check if path is a local drive
+        if platform.startswith('win') and len(Path(location).parents) == 0:
```

Comment:
Why are we skipping this check on windows?

Reply:
This function with `ctypes.windll.kernel32.GetFileAttributesW` retrieves file system attributes for a specified file or directory. It returns True for C:\, D:\, ...  But in fact, these drives are not hidden. To count the files you always need to use, the `--all` argument.